manifests/kustomize/base/webhook/pipelineversion-cert.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: cert-manager.io/v1
```

Comment:
It doesn't seem like this certificate is used on the API server deployment. Also, it seems like the cache server (a v1 compatibility thing) leverages cert-manager when installed from github.com/kubeflow/manifests, so we can probably do something similar by adding the cert-manager manifests at `manifests/kustomize/env/cert-manager/base`.